filament/include/filament/MorphTargetBuffer.h

```diff
@@ -112,12 +112,11 @@ class UTILS_PUBLIC MorphTargetBuffer : public FilamentAPI {
      *
      * @param engine Reference to the filament::Engine associated with this MorphTargetBuffer.
      * @param targetIndex the index of morph target to be updated.
-     * @param tangents pointer to at least count tangents
-     * @param count number of tangent elements in tangents
-     * @see setTangentsAt
+     * @param normals pointer to at least count normals
+     * @param count number of tangent elements in normals
      */
-    void setTangentsAt(Engine& engine, size_t targetIndex,
-            math::short4 const* tangents, size_t count, size_t offset = 0);
+    void setNormalsAt(Engine& engine, size_t targetIndex,
+            math::short4 const* normals, size_t count, size_t offset = 0);
```

Comment:
The reason we're passing a short4 it's because it's a tangent frame encoded as a quaternion. If you are now passing just normals… why is it a short4?